dm/master/scheduler/scheduler.go

```diff
@@ -159,6 +159,7 @@ func NewScheduler(pLogger *log.Logger, securityCfg config.Security) *Scheduler {
 }
 
 // Start starts the scheduler for work.
+// NTOE: for logic errors, it should start without returning errors (but report via metrics or log) so that the user can fix them.
```

Comment:
```suggestion
// NOTE: for logic errors, it should start without returning errors (but report via metrics or log) so that the user can fix them.

```